src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import SearchResults from "../components/SearchResults";
+import { render, screen } from "@testing-library/react";
+
+describe("SearchResults", () => {
+  const validProps = {
+    results: [
+      {
+        data: [
```

Comment:
Convention with validProps data is to always use values like `test title`, `test id` etc. We shouldnt be using data pulled in from outside such as links like here